tests/integration/test_bits.py

```diff
@@ -21,67 +21,191 @@
 import cunumeric as num
 
 
-@pytest.mark.parametrize("ndim", range(1, LEGATE_MAX_DIM + 1))
-@pytest.mark.parametrize("dtype", ("B", "i", "?"))
-@pytest.mark.parametrize("bitorder", ("little", "big"))
-def test_packbits(ndim, dtype, bitorder):
-    in_np = np.array([], dtype=dtype)
-    in_num = num.array([], dtype=dtype)
-    out_np = np.packbits(in_np, bitorder=bitorder)
-    out_num = num.packbits(in_num, bitorder=bitorder)
-    assert np.array_equal(out_np, out_num)
-
-    for extent in (3, 5, 8, 16):
-        shape = (extent,) * ndim
+class TestPackbits(object):
+    def test_none_arr(self):
+        # Numpy raises "TypeError:
+        # Expected an input array of integer or boolean data type"
+        # For cuNumeric raises:
+        #  > if a.dtype.kind not in ("u", "i", "b"):
+        #  E AttributeError: 'NoneType' object has no attribute 'dtype'
+        with pytest.raises(AttributeError):
+            num.packbits(None)
+
+    def test_dtype(self):
+        shape = (3, 3)
+        in_num = num.random.random(size=shape)
+        msg = r"type"
+        # TypeError: Expected an input array of integer or boolean data type
+        with pytest.raises(TypeError, match=msg):
+            num.packbits(in_num)
```

Comment:
```suggestion
        # TypeError: Expected an input array of integer or boolean data type
        with pytest.raises(TypeError, match):
            num.packbits(in_num)
```
Per https://gitlab-master.nvidia.com/groups/legate/-/wikis/NumPy-compatibility-strategy#matching-numpy-behavior was decided not to match exception messages, only exception types (here and below)

Reply:
Got it and fixed.